I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 99, in script call
    call a1s1
  File "game/act1Scene1.rpy", line 33, in script
    play sound wood volume 0.2
  File "renpy/common/000statements.rpy", line 300, in execute_play_sound
    renpy.sound.play(_audio_eval(p["file"]),
  File "renpy/common/000statements.rpy", line 32, in _audio_eval
    return eval(expr, locals=store.audio.__dict__)
  File "game/act1Scene1.rpy", line 33, in <module>
    play sound wood volume 0.2
NameError: name 'wood' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 99, in script call
    call a1s1
  File "game/act1Scene1.rpy", line 33, in script
    play sound wood volume 0.2
  File "C:\Users\djkni\Downloads\renpy\renpy-7.4.11-sdk/renpy/ast.py", line 2015, in execute
    self.call("execute")
  File "C:\Users\djkni\Downloads\renpy\renpy-7.4.11-sdk/renpy/ast.py", line 2003, in call
    return renpy.statements.call(method, parsed, *args, **kwargs)
  File "C:\Users\djkni\Downloads\renpy\renpy-7.4.11-sdk/renpy/statements.py", line 278, in call
    return method(parsed, *args, **kwargs)
  File "renpy/common/000statements.rpy", line 300, in execute_play_sound
    renpy.sound.play(_audio_eval(p["file"]),
  File "renpy/common/000statements.rpy", line 32, in _audio_eval
    return eval(expr, locals=store.audio.__dict__)
  File "C:\Users\djkni\Downloads\renpy\renpy-7.4.11-sdk/renpy/python.py", line 2276, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "C:\Users\djkni\Downloads\renpy\renpy-7.4.11-sdk/renpy/python.py", line 2269, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/act1Scene1.rpy", line 33, in <module>
    play sound wood volume 0.2
NameError: name 'wood' is not defined

Windows-10-10.0.22000
Ren'Py 7.4.11.2266
Echoes That Sing 1.0
Tue Mar 29 09:24:35 2022
